32blit-stm32/Src/SystemMenu/SystemMenuController.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "persistence.h"
+
+Pen get_menu_colour(int index);
```

Comment:
A bit odd the declaration being here when the implementation is in 32blit.cpp.

Reply:
I will try to clean this up in a later update / separate PR. (was discussed on Discord that some cleanup might be good)